less/home-page.less

```diff
@@ -24,9 +40,23 @@
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
+
+      .blast {
+        @media @phone {
+          width: 100%;
```

Comment:
what are you selecting inside the media query @media @phone {}? 

what is width:100% applying to?